### PR TITLE
add link rel canonical for release url

### PR DESCRIPTION
**Pull request by toddjordan**
Initial pass at adding link canonical tag to api docs in support of `release` url
Part of addressing #407

### CODE REVIEW

File: app/templates/head.hbs [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <title>{{model.title}}</title>
 
 <link rel="dns-prefetch" href="{{cdnDomain}}" >
```
Comment (toddjordan) [inline app/templates/head.hbs]: actually this may need a bug from before, and need to be model.cdnDomain.  will check and update if necessary.

Comment (sivakumar-kailasam) [reply app/templates/head.hbs]: Yeah, what u said 😄

**Comment by sivakumar-kailasam** [comment]
Can we add a test for this?

**Comment by toddjordan** [comment]
yeah, will see what I can do for a test.  Should be able to assert on it with an acceptance test

**Pull request merged**